apps/meteor/ee/app/license/server/getStatistics.ts

```diff
@@ -31,39 +31,42 @@ export async function getStatistics(): Promise<ENTERPRISE_STATISTICS> {
 			statistics.seatRequests = count;
 		}),
 	);
-	// Number of livechat tags
-	statsPms.push(
-		LivechatTag.col.count().then((count) => {
-			statistics.livechatTags = count;
-			return true;
-		}),
-	);
 
-	// Number of canned responses
-	statsPms.push(
-		CannedResponse.col.count().then((count) => {
-			statistics.cannedResponses = count;
-			return true;
-		}),
-	);
+	if (hasLicense('livechat-enterprise')) {
```

Comment:
Can you extract this into its own function and just call it when license is there? (I assume this is since models are inside an onlicense now, so it's required)

Reply:
Sure, its done :heavy_check_mark: